renku/cli/exception_handler.py

```diff
@@ -59,6 +59,8 @@
 
 import click
 
+from renku.core.errors import RenkuException
+
 _BUG = click.style(
     'Ahhhhhhhh! You have found a bug. 🐞\n\n',
```

Comment:
Should we also update the message to factor in the feedback we received in Bern? Some humble message which is not mocking @ciyer would do the trick (we can also open a ticket for that).

Reply:
I think it deserves to have its own story.